internal/cli/quickstarts.go

```diff
@@ -465,3 +344,54 @@ func defaultCallbackURLFor(s string) string {
 func defaultURL(url string, port int) string {
 	return fmt.Sprintf("%s:%d", url, port)
 }
+
+func (i *qsInputs) fromArgs(cmd *cobra.Command, args []string, cli *cli) error {
```

Comment:
This is useful, could be an interface method in the future.

Reply:
👍🏻  I was thinking that as well, on our code health milestone we could invest into ensuring commands use a similar pattern enforced by an interface.